front/src/applications/stdcmV2/components/StdcmTableResults.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useState } from 'react';
+
+import { Button } from '@osrd-project/ui-core';
+import { PDFDownloadLink } from '@react-pdf/renderer';
+import { useTranslation } from 'react-i18next';
+
+import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import SimulationReportSheetV2 from 'applications/stdcm/components/SimulationReportSheetV2';
+import type { SimulationReportSheetProps, StdcmV2SuccessResponse } from 'applications/stdcm/types';
+import { getOperationalPointsWithTimes, getStopDurationTime } from 'applications/stdcm/utils';
+import type { RollingStockWithLiveries } from 'common/api/osrdEditoastApi';
+
+type SimulationTableProps = {
+  stdcmData: StdcmV2SuccessResponse;
+  pathProperties?: ManageTrainSchedulePathProperties;
+  rollingStockData: RollingStockWithLiveries;
+  speedLimitByTag?: string;
+  creationDate?: Date;
+  simulationReportSheetNumber: string;
+  mapCanvas?: string;
+  setIsSimulationSelected: (simulationSelected: boolean) => void;
+  isSimulationSelected: boolean;
+};
+
+const StcdmTableResults = ({
+  stdcmData,
+  pathProperties,
+  rollingStockData,
+  speedLimitByTag,
+  creationDate,
+  simulationReportSheetNumber,
+  mapCanvas,
+  setIsSimulationSelected,
+  isSimulationSelected,
+}: SimulationTableProps) => {
+  const { t } = useTranslation('stdcm-simulation-report-sheet');
+
+  const simulationReport: SimulationReportSheetProps = {
+    stdcmData,
+    pathProperties,
+    rollingStockData,
+    simulationReportSheetNumber,
+    speedLimitByTag,
+    creationDate,
+  };
+
+  const opList = getOperationalPointsWithTimes(simulationReport);
```

Comment:
To generate the op List, you only need the suggestedOperationalPoints (from pathProperties), the simulation and the departureTime (from stdcmData).
Modify the function to take only these 3 arguments, and you can clean the code like this:
```suggestion
  const opList = useMemo(() => getOperationalPointsWithTimes(
    pathProperties?.suggestedOperationalPoints || [],
    stdcmData.simulation,
    stdcmData.departure_time
  ), [pathProperties, stdcmData]);
```
Also add a useMemo to memoize it

Reply:
done